res/code/bml.php

```diff
@@ -139,11 +130,10 @@ function getutf8($s) {
   
   echo <<<END
 (function() {
-  //console.log('path');
   $kmwbase.init({
-    root: "//{$StaticResourceDomain}/kmw/engine/{$build}/", 
-    resources: "//{$StaticResourceDomain}/kmw/engine/{$build}/", 
-    keyboards: "//{$StaticResourceDomain}/keyboard/",
+    root: "https://{$StaticResourceDomain}/kmw/engine/{$ver}/", 
```

Comment:
We should always use `https`, regardless of current page protocol (e.g. this allows KMW BML to work on `file://` pages as well)